### PR TITLE
Miden M2 delivery

### PR DESCRIPTION
**Pull request by stratisnik**
Submission of 5th delivery

# Grant Application Checklist

- [ Y] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [ Y] An Ethereum address for the payment of the milestones is provided inside the application.  
- [ Y] The total funding amount of the project is up to USD $30k for every grant applications.
- [Y ] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

**Comment by AgungPrabowo123** [comment]
> Submission of 5th delivery
> 
> # Grant Application Checklist
> * [ Y] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
> * [ Y] An Ethereum address for the payment of the milestones is provided inside the application.
> * [ Y] The total funding amount of the project is up to USD $30k for every grant applications.
> * [Y ] The initial PR contains only one commit (squash if needed before submitting your PR).
s

**Comment by NecoSherry** [comment]
Hi, @stratisnik ! You have submitted the delivery for milestone 5 this time. However, you had previously only submitted the delivery for milestone 1. We usually review deliveries in sequential order by milestones. Therefore, we recommend that you submit the deliveries for milestones 2, 3, and 4 before submitting the one for milestone 5.

**Comment by stratisnik** [comment]
> Hi, @stratisnik ! You have submitted the delivery for milestone 5 this time. However, you had previously only submitted the delivery for milestone 1. We usually review deliveries in sequential order by milestones. Therefore, we recommend that you submit the deliveries for milestones 2, 3, and 4 before submitting the one for milestone 5.

Hi @NecoSherry, We faced some issues during the submission of the previous milestone resulting in funds being lost. This issue has already been communicated with the veChain Grant. As such we found it very difficult to proceed with milestones 2,3 and 4. I would appreciate it if you could look into the submission of this milestone and release the funds for one of the deliverables so we can proceed with the project. Thank you

**Comment by NecoSherry** [comment]
Hi, @stratisnik , after completing milestone 5 , what are your next steps? Do you intend to move milestone 5 before milestone 2?

**Comment by stratisnik** [comment]
> Hi, @stratisnik , after completing milestone 5 , what are your next steps? Do you intend to move milestone 5 before milestone 2?

Hi @NecoSherry. Yes with the release of the funds, the aim is to proceed then with milestones 2,3,4.

**Comment by NecoSherry** [comment]
@stratisnik We recommended you update the milestones in the development plan. Then we will proceed with delivery and payment based on the updated sequence of milestones.

**Comment by stratisnik** [comment]
@NecoSherry sequence of milestones updated here: Update miden.md - reordered sequence of milestones #252

File: milestone-delivery/deliveries/milestone-delivery/deliveries/miden-5.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+> Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. Don't remove any of the mandatory parts presented in bold letters or as headlines!
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:** https://github.com/vechain/grant-program/pull/135
+* **Milestone Number:** 5
```
Comment (libotony) [inline milestone-delivery/deliveries/milestone-delivery/deliveries/miden-5.md:8]: Could you updating the milestone number accordingly? It should be Milestone 2 after the adjustment.

Comment (stratisnik) [reply milestone-delivery/deliveries/milestone-delivery/deliveries/miden-5.md]: @libotony updated as requested. See commit 5c70dae217c0d8a0d5ebb5c13f18f90eb7622a62

**Review by libotony — APPROVED** [review]

**Review by NecoSherry — APPROVED** [review]

**Comment by NecoSherry** [comment]
Thank you for the delivery. We will promptly proceed with the payment process for milestones 3. Before we make the payment, could you please confirm if the payment address is still the same as in your application?

**Comment by stratisnik** [comment]
Hi NecoSherry,
The address is:
0xFE59E858d5999C16A6f60bf521209167Abf6Df15

which is the address that I see here
<https://github.com/vechain/grant-program/blob/master/applications/miden.md> on
the application.

Thank you.

**Comment by stratisnik** [comment]
Hi @NecoSherry 

The address is:
0xFE59E858d5999C16A6f60bf521209167Abf6Df15

which is the address that I see [here](https://github.com/vechain/grant-program/blob/master/applications/miden.md) on the application.

Thank you.

**Comment by NecoSherry** [comment]
Thank you for confirming the address. The payment request has been initiated, and it will take some time to process. We appreciate your patience in advance.

**Pull request merged**